components/GoalsAndMission/GoalsAndMission.js

```diff
@@ -0,0 +1,19 @@
+import { Container, Row, Col } from "../bootstrap"
+import AboutPagesCard from "../AboutPagesCard/AboutPagesCard"
+import GoalsAndMissionCardContent from "../OurGoalsAndMissionCardContent/OurGoalsCardContent"
+import styles from "../GoalsAndMission/GoalsAndMission.module.css"
+
+const GoalsAndMission = () => {
+  return (
+    <Container fluid className={styles.container}>
+      <Row>
+        <Col>
+          <h1 className="mt-5 fw-bold">Our Goals and Mission</h1>
```

Comment:
Should check with @jamesvas5307 if all headings should use bold typeface. If so we can set globally